code/datums/quirks/negative.dm

```diff
@@ -794,6 +794,9 @@
 	if(IS_IN_STASIS(quirk_holder))
 		return
 
+	if(quirk_holder.stat == DEAD)
+		return
```

Comment:
what about chems that work in dead people?

Reply:
I can't think right now of any chem that works on dead people and simultaneously _doesn't_ remain on their bloodstream after they're revived, so we still get the allergic reaction when you get revived but you don't end up with 500 histamines while dead for no reason.